Dokan/Dokan/Scenes/ProductDetails/Cell/ReviewerTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReviewerTableViewCell.swift
+//  Dokan
+//
+//  Created by raniazeid on 28/07/2022.
+//
+
+import UIKit
+
+class ReviewerTableViewCell: UITableViewCell {
+
+   @IBOutlet weak var reviewerImageView: UIImageView!
+    
+    @IBOutlet weak var reviewerNameLabel: UILabel!
+    
+    @IBOutlet weak var reviewerComment: UITextView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    //MARK: - method to configure cell
```

Comment:
- Leave white spaces before Mark
- Follow sentence cases, first letter is capitalized. 
```suggestion
    
    // MARK: - Method to configure cell
```